### PR TITLE
Complete set of partitioned schemas for TPCDS

### PR DESCRIPTION
**Pull request by lmcarril**

### CODE REVIEW

File: benchmarks/tpcds/schemas/psql_native_partitioned_id_hash/indexes.sql [hunk 1]
```diff
@@ -0,0 +1,167 @@
+--create index idx_customer_demographics_1      on customer_demographics (cd_marital_status,        cd_education_status);
```
Comment (Spiros-N-Agathos) [inline benchmarks/tpcds/schemas/psql_native_partitioned_id_hash/indexes.sql:1]: Remove the comments?

**Review by Spiros-N-Agathos — APPROVED** [review]

**Comment by Spiros-N-Agathos** [comment]
Checking the diffs, you just mixed-and-matched what was already in the repo to produce new schemas right?

File: benchmarks/tpcds/schemas/psql_native_partitioned_id_hash/indexes.sql (hunk 1 above)
Comment (lmcarril) [reply benchmarks/tpcds/schemas/psql_native_partitioned_id_hash/indexes.sql:1]: Are the same comments as in any other index file in all TPCDS schema, I prefer to leave them there so all index.sql files are as similar as possible.

**Comment by lmcarril** [comment]
> Checking the diffs, you just mixed-and-matched what was already in the repo to produce new schemas right?

Yes exactly, that is the philosophy behind most schemas, ideally we would only need to add/remove file between schemas variations, but it is not possible in all cases.

**Pull request merged**